sdwan_feature_templates.tf

```diff
@@ -2014,3 +2014,288 @@ resource "sdwan_security_app_hosting_feature_template" "security_app_hosting_fea
+  interface_name_variable          = try(each.value.interface_name_variable, null)
+  interface_description            = try(each.value.interface_description, null)
+  interface_description_variable   = try(each.value.interface_description_variable, null)
+  shutdown                         = try(each.value.shutdown, null)
+  shutdown_variable                = try(each.value.shutdown_variable, null)
+  tunnel_source_interface          = try(each.value.tunnel_source_interface, null)
+  tunnel_source_interface_variable = try(each.value.tunnel_source_interface_variable, null)
+  tunnel_source                    = try(each.value.tunnel_source_ip, null)
+  tunnel_source_variable           = try(each.value.tunnel_source_ip_variable, null)
+  tunnel_destination               = try(each.value.tunnel_destination, null)
+  tunnel_destination_variable      = try(each.value.tunnel_destination_variable, null)
+  ip_address                       = try(each.value.ip_address, null)
+  ip_address_variable              = try(each.value.ip_address_variable, null)
+  ip_mtu                           = try(each.value.ip_mtu, null)
+  ip_mtu_variable                  = try(each.value.ip_mtu_variable, null)
+  tcp_mss_adjust                   = try(each.value.tcp_mss, null)
+  tcp_mss_adjust_variable          = try(each.value.tcp_mss_variable, null)
+  clear_dont_fragment              = try(each.value.clear_dont_fragment, null)
+  clear_dont_fragment_variable     = try(each.value.clear_dont_fragment_variable, null)
+  rewrite_rule                     = try(each.value.rewrite_rule, null)
+  rewrite_rule_variable            = try(each.value.rewrite_rule_variable, null)
+  tracker                          = try([each.value.tracker], null)
+  tracker_variable                 = try(each.value.tracker_variable, null)
+  application                      = try(each.value.application, null)
+  application_variable             = try(each.value.application_variable, null)
+  access_lists = try(each.value.ipv4_ingress_access_list, each.value.ipv4_ingress_access_list_variable, each.value.ipv4_egress_access_list, each.value.ipv4_egress_access_list_variable, null) == null ? null : flatten([
+    try(each.value.ipv4_ingress_access_list, each.value.ipv4_ingress_access_list_variable, null) == null ? [] : [{
+      acl_name          = try(each.value.ipv4_ingress_access_list, null)
+      acl_name_variable = try(each.value.ipv4_ingress_access_list_variable, null)
+      direction         = "in"
+    }],
+    try(each.value.ipv4_egress_access_list, each.value.ipv4_egress_access_list_variable, null) == null ? [] : [{
+      acl_name          = try(each.value.ipv4_egress_access_list, null)
+      acl_name_variable = try(each.value.ipv4_egress_access_list_variable, null)
+      direction         = "out"
+    }]
+  ])
+}
+
+resource "sdwan_vpn_interface_cellular_feature_template" "vpn_interface_cellular_feature_template" {
```

Comment:
While testing this I discovered a bug in provider. To reproduce:
1. Create this resource using TF
2. Go to UI, click Edit and Save. Nothing will happen, in console you will see an error "e.ip is undefined"

This is probably because the provider doesn't include the mandatory, non-configurable block in JSON payload that should look like this:
<img width="143" alt="Screenshot 2024-12-02 at 10 41 37" src="https://github.com/user-attachments/assets/7b177c19-d41d-452c-89a1-7cf9a4341f33">

Reply:
thanks @tzarski0 

i opened a provider issue. https://github.com/CiscoDevNet/terraform-provider-sdwan/issues/378

Reply:
this is fixed.